html/templates/ontime/ontime.php

```diff
@@ -108,6 +108,21 @@
                 <div class="clear"></div>
             </div>
         </div>
+        <?php
+        if ($envWarning) {
```

Comment:
Please declare `$envWarning` with default value `false` or add an `isset` check here as it will trigger an notice error in php.